src/test/java/ai/elimu/LetterToAllophoneMappingsControllerTest.java

```diff
@@ -1,22 +1,21 @@
-package dev.ai.elimu.rest;
+package ai.elimu;
```

Comment:
This test should be placed in the same package as the class being tested (i.e. `ai.elimu.rest.v2.content`)?